fastf1/testing/reference_values.py

```diff
@@ -26,14 +27,20 @@ def ensure_data_type(column_dtypes, df):
               'SpeedI2': 'float64',
               'SpeedFL': 'float64',
               'SpeedST': 'float64',
+              'IsPersonalBest': 'bool',
               'Compound': 'O',
               'TyreLife': 'float64',
               'FreshTyre': 'bool',
-              'LapStartTime': '<m8[ns]',
               'Team': 'O',
-              'Driver': 'O',
+              'LapStartTime': '<m8[ns]',
+              'LapStartDate': '<M8[ns]',
               'TrackStatus': 'O',
-              'IsAccurate': 'bool'}
+              'Position': 'float64',
+              'Deleted': 'bool',
```

Comment:
Good catch noticing the missing values here. But 'Deleted' needs to be 'O' as well, because that column is optional. Therefore, None values need to be supported which requires object-dtype instead of bool. This is a bit unfortunate but there's no simple way arround that.

Reply:
This change is causing test failures now

`TypeError: Dtype ('Deleted', dtype('bool')) not equivalent to target dtype O`

Seems like both types are possible. Skip checking this column?

Reply:
Ok, this was not the end of this, apparently. Now, some tests fail with the inverse problem. I'd say, this is out of scope for this PR. Please comment the 'Deleted' line that's causing the failure and add a "TODO: fix test" comment. It's not worth holding this up any longer.